src/main/webapp/scripts/user-script.js

```diff
@@ -118,14 +119,53 @@ function getAllUsers(){
         }
         }).
         then(response => response.json())
-        .then((ous) => {
-            console.log(ous);
+        .then((userJSON) => {
+            let users = userJSON['users'];
+            for(var i = 0; i < users.length; i++){
+                let user = users[i];
+                var fullname = user['name']['fullName'];
+                var id = user['id'];
+                var orgUnitPath = user['orgUnitPath'];
+                var userJSON = {"name": fullname, "id": id, "orgUnitPath": orgUnitPath};
+                DFSTree(data, orgUnitPath, userJSON);
+            }
+            addValue(data);
+            visualize();
         })
         .catch((error) => {
             console.error('Error:', error);
         });
 }
 
+// add user to the OU it's in by DFS 
+function DFSTree(node, path, user){
```

Comment:
I think this function should be renamed to what it does rather than how it does it. Maybe rename to addUsertoOUByPath  or something else?